composer.json

```diff
@@ -13,7 +13,7 @@
         "rss": "https://github.com/laminas/laminas.dev/releases.atom"
     },
     "require": {
-        "php": "~8.2.0",
+        "php": "~8.3",
```

Comment:
Note that this also includes `"php": "8.4"`: intentional?

Reply:
No. I was hurrying to test platformsh build and got sloppy.